File: benchmark/python/array_push_pop.py
n = 50000
arr = []
for i in range(1, n + 1):
    arr.append((i * 19) % 997)

total = 0
for _ in range(1, n + 1):
    total = (total + arr.pop()) % 1_000_000_007
print(total)

File: benchmark/rust/array_push_pop.rs
fn main() {
    let n: usize = 50_000;
    let mut arr = Vec::new();
    for i in 1..=n {
        arr.push(((i as i64) * 19) % 997);
    }

    let mut total: i64 = 0;
    for _ in 1..=n {
        total = (total + arr.pop().unwrap()) % 1_000_000_007;
    }
    println!("{total}");
}
